demos/nrf52-beacon/.cargo/config

```diff
@@ -2,7 +2,7 @@
 target = 'thumbv7em-none-eabi'
 
 [target.'cfg(all(target_arch = "arm", target_os = "none"))']
-runner = 'arm-none-eabi-gdb'
+runner = 'arm-none-eabi-gdb -q -x ../openocd.gdb'
```

Comment:
This was intentional, since it allows using a different GDB script without having having a dirty git status because of it. You're supposed to copy `openocd.gdb` to `.gdbinit`, but it seems this isn't documented anywhere.

Reply:
Ah, okeis. Reverted.